Documentation/deploy-master.md

```diff
@@ -81,6 +81,34 @@ Again, we will use a [systemd drop-in][dropins]:
 [Unit]
 Requires=flanneld.service
 After=flanneld.service
+[Service]
+EnvironmentFile=/etc/kubernetes/cni/docker_opts_cni.env
+```
+
+Create the Docker CNI Options file:
+
+**/etc/kubernetes/cni/docker_opts_cni.env**
+
+```yaml
+DOCKER_OPT_BIP=""
+DOCKER_OPT_IPMASQ=""
+```
+
+If using Flannel for networking. 
+
+*Note:* Do not use below if you intend to use Callico for networking
```

Comment:
I try not to use "Note:" type stuff. Is there a way to rephrase, maybe combining with the "If using flannel" comment?